kustomize/overlays/prod/manager/keycloak-realm.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
xiJTQQMTV7bvSZaeRsdkY/cCSoKYlNRaF9ZEfg54EqT0e33phK/LIgzgCvUYHZajpivn2W5jocRx88vaCqi/ZiXqrUmhCHQTcTFh1ATaB6WRXWnvnz8rs5WYAOqRJz9tKs1VYufswFgURqOHvIur/67mjpTG6X4pxQgrgGEEjRxQXlp6RcmUPk9r8TdXakzs4aSAsxOdZ3K9azyYvbliSwzWXDCKIsDf2c1y6IETkVziIP8z/6YWu+52L0HKXr0396efPCve0CteHjrC4Nso3VPjGXy2cG5R5HcjP1LaRFNkZzihTjUW6Dj3nj2ABFLfzIyx9at8sqfUm1r8BAmXmxFPQ+8AzkSnYqdEqNcL19jxAMP1VTngOVCahKUbShTzE/YzxqaZnbIUfXdPZfX7Rj9BWDZ8WQEqaEdzZmoAz/JR/dHqBxTEwZHrwUB6hdakCJ3YWd1r1D+MsqjQzQoA4bPUJvWCKYpEsHWprbu/OUdnqzZQGTDt4qZjQK1YX07cT4vLsZvvxfuv0MLc6E4qcLxD5JWQXzIJdw997Iedbuqx7/rXmYtKXo0S54kGSWZQOvDKDs7dbaYMK53j03KEAkEU7DydvrFLEmbslSnAdlKI2CAR38WKOXBCYJW6f6Jq0PgxEnL7/tqpC2AStdmmSU0e1KMYMG4XXKv7He7fZ3uaXm4qdC9hd8xiH6/4cMhQz+j9bTwZMuO5SEqT51deWxTuT793pByKlj3r796JL/sKufYYHHaIjMgtQD+nQ7rNNVtG7OTlTranQqlUzsvNAHhBKQl9LyDGjbcB9E+lJQIZsqeM4jxRc4N1q3HBWXHUa8uxrAeTt15E3laCr3MKpM1f/JA+5CKvVKYN6ckj5tWzjx/Vz7c+Jig+pur/AN/hIYvBGxB88gTQSYhySRhWI8L6qYPld4HJ4U9dn8LbWXO301Fxp37xPFhL01xxFCGOfy90XuVyze5bhVu99Dc4E5zIOS6S2w5qJemu1sasDPa9aGgO5W/G2Pe0+MWeGT2A776XfO8uIz2VB2JOwqTIvFf8czzc0IVRFYgbnAp0rq2PGOkzanIuxjUAYPLs2gSp2m9Zm0Y1KZtt0fIYlhjc6HfMwzYjVKnUE/JMw+UjBxOY2aH4lHBN+IGbT6u+hJ0SNxZUy5Lzf+MCfgZA9GdUD2iLZBOLbyZ0yO3kwwTKq+zsFLClLoz7mjoYkhtOvl4ylCkxINnspt3JlldNSshi4fW6268De6A9vmn7PuQKQK7FrGGSeoSbY6JctKajJ1QZZFYqLTq30UFQ/DMlS/WWmU1fIDvap+QOpsMmfX2sU/auPcon69o3UeXYQnBMP6A4yaOCe+Ou1OHLmt/LXajUBqmm010namS+6pI6FIHMPrkKrVrtKRLZE/CvwGuSPaI91Fg9r4uA+h/8QP4e08bG9vxvu533SbwC5NHDxz35ZVgGcK27VoLzKbTD+utnR3rdMgRDJGvV/j/ZgAeyxKhOoSGL5OOUTYFMrMt4clY9DAQqscZ5zP4r8cUo+hQAwenRJi3RiXgv/yTyfgxt+HQ7y6qAJQ2TMLTmWPi4S+I5M+gkJfWyWcw3pZvG7J/VUOWTGwtH/RcwWkfnxv1uEHvuAo1NzDT2xmrXsXOxC5/zlwgGnQYRsrwcA3j9+gY/9sD2LJ4Wfwsa2LuzKxlj34U4rFioBT1gIHRJpwd7xifKAVxWBqM8/BZpL5fWqYmER7ac/7E8NqhDxdm3ItdDt15tMdDp7iy2iARaStIXUMiDPTCjSnqH79/TTDl0HoxwWhW+nCIxPZ2vhUov4OZ77LMLX2uJTFTg2S7YuDsyKcT4p5nsmvtn3/zY58tpl0w8s3QT2PNbhyPrb64+msgg0TY0bDxbIYnW2T9QQFM6fh0CGvA+Gav5QL6uvCzrByxjZeDzybh16w0DoG/R93EVOsrjtRrgFTVlPDUbAeaXVJy6YPNOE6vMZyLCkoa27A5NdFpKNYmwiqeZRDgDCkOIfya+DFNNHKi6eRg7AyWt2Na6o4lCECsEh+GID0G3Ugz1G+UucXlnMi9/8mp8UFLRp+d4OtfWa61rHV0MGe7fXBW7twV9J1qSvWbS4S7GFQqilPdtG0PZhZWjeaHe9bACkKQkrK059+L+q3BpllXR2h7Ye05JTyQPTFXso2u9b4Z7DouxQ/a/nHkCBYT4e66Mbg6Yuc/13u80B3VPetJngH3h9qen/inSxQUDRryrl2TUwiuwbneP2DHmuWLp6yIrs0IeGZppLOzd3GvsrC31G8shD+lEVxtgPX2dE0C6PrzeTOpai8b+1f1oPLW+bhRBkpkI+d8DnJAMcEeBM6EXTPXrF1XMdFZiuakbDnvOdTTmonJGnK1PA6sX0Tjp2Q/YohQtVi1oZsqa9/6xYbV4Kr6+xM1jur64LSlLG8wtwlZhOiNnsatEbrAkryBe1AUULiWiGipL5rqWQ9OjO6YwK2E3PHaCWZYu659+4r3iCIIaPMukCrA0W3CkkSvjL2x677tFNpjIix0n+NPJBfJAzshlUSDE7vA6GsEtfLsHhm5mxNHjpBb7tbP4kduyvC6dP9eMdLzp1au62Q4T2JUp4XReTRrqPhtBWa/YzRE7U7TRiXBY7VOR1RDyTDskFf7wkwOQ+1jlG8xJKiiTiMBFj8rFWR6vLBHhqMuOuIXe9jJH9qcgwSliGofXp04fU2tTU0VLtw6iW4k+Pvc/zUvhwuQO/zelnLIYQdgOb70AjUmX+WZ/0gJ5ARUyMsJueJ1Rp2tOM0AarXxD7B7lZXEiiy/xhcBFlUuC85mkmsDdaUgLdhza3umJMMpBD9rJLW02ixEz03ZWX9ZWOIKyiPdGxAn+EpYwV1LytrJTd8bzRo+XDvSOUaXhfe+KcBJ+A/FvDQYI5MpHr7bD3Od+u/vjF7ELAUf8rX3TvJgLal7p2j5zNCPNMj6XeonDML7SQdQIGHEgj2c8AyFfi0zUPp7c/mvgXGlC9cjtHur3DQ==
```

Comment:
I have ensured that our demo environment credentials are the same as before, but this now includes the extended realm configuration from the Keycloak export.